eve/workers/unit_and_feature_tests/supervisor/buildbot_worker.conf

```diff
@@ -0,0 +1,4 @@
+[program:buildbot_worker]
+command=/bin/sh -c 'buildbot-worker create-worker . "%(ENV_BUILDMASTER)s:%(ENV_BUILDMASTER_PORT)s" "%(ENV_WORKERNAME)s" "%(ENV_WORKERPASS)s"  && buildbot-worker start --nodaemon'
+autostart=true
+autorestart=false
```

Comment:
missing newline

Reply:
Where exactly should we have the newline?

Reply:
A newline at the very end of the file (no big deal just noticing and github diff shows it with a special symbol).